Helper_Scripts/TTS_Installers/install_tts_pocket_tts_onnx.py

```
@@ -0,0 +1,465 @@
+        repo_root: Repository root to relativize against.
+
+    Returns:
+        A POSIX-style relative path when possible, otherwise an absolute path.
+    """
+    if not path.is_absolute():
+        return path.as_posix()
+    if repo_root:
+        try:
+            return path.relative_to(repo_root).as_posix()
+        except Exception:
+            pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)